core-icon.html

```diff
@@ -167,7 +167,11 @@
         }
 
         this.iconset = this.computeIconset(this.icon);
-        this.iconsetElement.applyIcon(this.root, this.iconName, this.theme);
+        if (this.iconsetElement) {
+          this.iconsetElement.applyIcon(this.root, this.iconName, this.theme);
+        } else {
+          console.warn('iconset "' + this.iconset + '" not found!');
```

Comment:
I discussed this with @ssorallen - this should really be an exception. Either `this.icon` is an empty string, which means we should never even get here, or the iconset specified isn't present, which seems like an error to me. What do you think?